src/database/models/userExtension.js

```diff
@@ -54,6 +54,10 @@ module.exports = (sequelize, DataTypes) => {
 				type: DataTypes.INTEGER,
 				allowNull: false,
 			},
+			external_mentee_visibility: {
+				type: DataTypes.STRING,
+				defaultValue: 'CURRENT',
+			},
```

Comment:
Check if we need mentee_visibility_policy here.
@Prajwal17Tunerlabs

Reply:
according to SAAS policy requirements, YES we need "external_mentee_visibility" field in userExtension as well as mentorExtension